pkg/eestream/secretbox_test.go

```diff
@@ -26,8 +26,8 @@ func TestSecretbox(t *testing.T) {
 		t.Fatal(err)
 	}
 	data := randData(encrypter.InBlockSize() * 10)
-	encrypted := TransformReader(bytes.NewReader(data),
-		encrypter, 0)
+	encrypted := TransformReader(
+		ioutil.NopCloser(bytes.NewReader(data)), encrypter, 0)
```

Comment:
and this?

Reply:
Yes, it's needed. `TransformReader()` now expects `io.ReadCloser`.